recipes/mingw-w64/linux/conanfile.py

```diff
@@ -307,10 +311,4 @@ def package_info(self):
         self.env_info.STRIP = prefix + "strip"
         self.env_info.GCOV = prefix + "gcov"
         self.env_info.RC = prefix + "windres"
-        # Symlinks cannot be created in package step, otherwise the link target is wrong.
```

Comment:
Wait the old comment said this was not possible 🤔

Reply:
It was not possible because the links were absolute.
To your question above, it does work consuming from an Artifactory server.